### PR TITLE
Transaction first step

### PR DESCRIPTION
**Pull request by Fkayyali**
### Jira URL

https://optimumpartners.atlassian.net/browse/JIRA_ISSUE_ID

### Description

_What problem is this branch meant to solve, what changes does it make, and how do these changes fix that problem?_

### Dependencies in Other PRs,Repo

_If this pull request relates to or depends on any changes in backend/frontend repo PR link it here._

### Testing Instructions

_How to test this and check for any possible regressions. Provide test data when applicable._

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -17,13 +17,14 @@
         "react-countdown-circle-timer": "^2.5.4",
         "react-datepicker": "^4.5.0",
         "react-dom": "^17.0.2",
-        "react-dropdown-select": "^4.8.2",
         "react-feather": "^2.0.9",
         "react-hook-form": "^7.22.3",
         "react-places-autocomplete": "^7.3.0",
         "react-router-dom": "^6.0.2",
         "react-scripts": "4.0.3",
+        "react-select": "^5.2.2",
```
Comment (AhmadAlShwaiki) [inline package-lock.json:25]: Please remove this file from this commit

File: src/assets/gray-mail.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="13" height="10" viewBox="0 0 13 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (AhmadAlShwaiki) [inline src/assets/gray-mail.svg:1]: I remember seeing some of these values already, like the agent picture, please make sure to remove any repeated assets.

File: src/components/transaction/AddInvolvedParties/add-involved-parties.styles.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+export const SellerTeamFromContainer = styled.div`
+  width: 100%;
+  height: 383px;
+  background-color: #fafafa;
+  margin: 42px 0 0 0;
+`;
+
+export const TitleText = styled.div`
+  font-size: 24px;
+  color: #222222;
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/add-involved-parties.styles.js:43]:
```suggestion
  color: colors.myColor;
```

File: src/components/transaction/AddInvolvedParties/add-involved-parties.styles.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+export const SellerTeamFromContainer = styled.div`
+  width: 100%;
+  height: 383px;
+  background-color: #fafafa;
+  margin: 42px 0 0 0;
+`;
+
+export const TitleText = styled.div`
+  font-size: 24px;
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/add-involved-parties.styles.js:42]:
```suggestion
  font-size: my-value rem;
```

File: src/components/transaction/AddInvolvedParties/index.jsx [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+  TextArea,
+} from './add-involved-parties.styles';
+
+/**
+ * Add Involved Parties.
+ *
+ * @return {JSX.Element} Transaction first page.
+ */
+export default function AddInvolvedParties() {
+  // const [selectedAgent, setSelectedAgent] = useState();
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/index.jsx:39]:
```suggestion
```

File: src/components/transaction/AddInvolvedParties/index.jsx [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+    { value: 'yousef', label: 'Yousef' },
+    { value: 'sara', label: 'Sara' },
+  ];
+  const buyerOptions = [
+    { value: 'faris', label: 'Faris' },
+    { value: 'ammar', label: 'Ammar' },
+    { value: 'tamam', label: 'Tamam' },
+    { value: 'yousef', label: 'Yousef' },
+    { value: 'sara', label: 'Sara' },
+  ];
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/index.jsx:74]:
```suggestion
 Can just render names and then change the first letter toUpperCase
```

File: src/components/transaction/AddInvolvedParties/index.jsx [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+  // const agentOptions2 = ['faris', 'ammar', 'tamam', 'yousef', 'sara'];
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/index.jsx:75]:
```suggestion
```

File: src/components/transaction/AddInvolvedParties/index.jsx [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+
+  const submit = async (data) => {
+    console.log(data);
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/index.jsx:78]:
```suggestion
```

File: src/components/transaction/AddInvolvedParties/index.jsx [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+  };
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    control,
+    //  getValues,
```
Comment (AhmadAlShwaiki) [inline src/components/transaction/AddInvolvedParties/index.jsx:86]:
```suggestion
```

**Review by AhmadAlShwaiki — APPROVED** [review]

**Review by mohdabed123 — APPROVED** [review]

**Review by IbrahimAldereni-socuimtech — CHANGES_REQUESTED** [review]

**Review by IbrahimAldereni-socuimtech — APPROVED** [review]

**Pull request merged**